src/interactions/owner/GivePoints.ts

```diff
@@ -65,18 +65,39 @@ export default class GivePoints extends BotInteraction {
                 placeholder.role = 'Placeholder';
                 existingPlaceholder = await repository.save(placeholder);
             }
-            const newData: any = [...Array(quantity)].map((element) => ({ participant: user.id, role: 'Placeholder', trial: existingPlaceholder }));
-            await dataSource
-                .createQueryBuilder()
-                .insert()
-                .into(TrialParticipation)
-                .values(newData)
-                .execute()
+            if (quantity > 0) {
+                const newData: any = [...Array(quantity)].map((element) => ({ participant: user.id, role: 'Placeholder', trial: existingPlaceholder }));
+                await dataSource
+                    .createQueryBuilder()
+                    .insert()
+                    .into(TrialParticipation)
+                    .values(newData)
+                    .execute()
+            }
+            else if (quantity < 0){
+                const participationRepo = dataSource.getRepository(TrialParticipation);
+                let participationToDelete = await participationRepo.find({
+                    where: {
+                        participant: user.id,
+                        // reaper: existingPlaceholder
+                    }
+                })
+                if (participationToDelete) {
+                    console.log('Placeholder entry found.');
+                    await participationRepo.remove(participationToDelete.slice(0, quantity-1));
```

Comment:
We shouldn't need this slice here. It's better to use a `LIMIT` on line 79 so that the load is handled in the database call rather than having to slice every time this command is used.

Reply:
It seems with the library we're using 'typeorm', 'limit' is instead 'take'. Updated code accordingly.